ldes-server-port-admin-rest/src/main/java/be/vlaanderen/informatievlaanderen/ldes/server/admin/rest/converters/ModelConverter.java

```diff
@@ -33,7 +40,7 @@ public boolean canWrite(Class<?> clazz, MediaType mediaType) {
 
 	@Override
 	public List<MediaType> getSupportedMediaTypes() {
-		return List.of(MediaType.ALL);
+		return List.of(DEFAULT_MEDIA_TYPE, MediaType.ALL);
```

Comment:
I see you did this everywhere except for ListViewHttpConverter is this forgotten or on purpose?